lib/client.js

```diff
@@ -378,8 +378,21 @@ class AssetComputeClient extends EventEmitter {
         }
         return response;
     }
-}
 
+    async isEventJournalReady(journalUrl) {
```

Comment:
btw, the journal url should already be in [`this.journal`](https://github.com/adobe/asset-compute-client/blob/aaad1837c9ab603608e9ac9b5b1617bd27495987/lib/client.js#L195) set in the register call. 

I would suggest maybe checking if the journal URL is defined in `this.journal` if no `journalUrl` parameter is passed in

Reply:
that way, the user does not need to store the journal URL outside of the `asset-compute-client`

Reply:
You could also set `journalUrl` to this.journal as `default` value.
Do you use `journalUrl` to simplify unit testing? If it is so, remember nothing is private in javascript (node10-12), and you could overwrite the object's `this.journal` in any test if you would need to. If `journalUrl` is supposed to be the same as `this.journal`, remove the param and use `this.journal` to use the object's state rather than params.

Reply:
thanks! will try it out